lib/secret.ts

```diff
@@ -49,3 +53,97 @@ export class SecretFromVariable extends Resource {
     return secret;
   }
 }
+
+export class SecretSet extends Construct {
```

Comment:
nit: We should rename the file to `secrets` or split it into multiple files that match the exported classes a bit better.